spec/support/devise.rb

```diff
@@ -0,0 +1,5 @@
+require 'rspec/rails'
```

Comment:
A+ use of the `spec/support` directory for keeping testing-related config organized.